src/foundry/foundry.js/localization.d.ts

```diff
@@ -1,123 +1,140 @@
-   *
-   * @example
-   * ```typescript
-   * {
-   *   "MYMODULE.GREETING": "Hello {name}, this is my module!"
-   * }
-   * game.i18n.format("MYMODULE.GREETING", {name: "Andrew"}); // Hello Andrew, this is my module!
-   * ```
-   */
-  format(stringId: string, data?: Record<string, any>): string;
+  class Localization {
+    constructor(serverLanguage?: string);
+
+    /**
+     * The target language for localization
+     * @defaultValue `'en'`
+     */
+    lang: string;
+
+    /**
+     * The package authorized to provide default language configurations
+     * @defaultValue `'core'`
+     */
+    defaultModule: string;
+
+    /**
+     * The translation dictionary for the target language
+     * @defaultValue `{}`
+     */
+    translations: Translations;
+
+    /**
+     * Fallback translations if the target keys are not found
+     * @defaultValue `{}`
+     */
+    protected _fallback: Translations;
+
+    /* -------------------------------------------- */
+
+    /**
+     * Initialize the Localization module
+     * Discover available language translations and apply the current language setting
+     * @returns A Promise which resolves once languages are initialized
+     */
+    initialize(): Promise<void>;
+
+    /* -------------------------------------------- */
+
+    /**
+     * Set a language as the active translation source for the session
+     * @param lang - A language string in CONFIG.supportedLanguages
+     * @returns A Promise which resolves once the translations for the requested language are ready
+     */
+    setLanguage(lang: string): Promise<void>;
+
+    /* -------------------------------------------- */
+
+    /**
+     * Discover the available supported languages from the set of packages which are provided
+     */
+    protected _discoverSupportedLanguages(): Record<string, string>;
+
+    /**
+     * Prepare the dictionary of translation strings for the requested language
+     * @param lang - The language for which to load translations
+     * @returns The retrieved translations object
+     */
+    protected _getTranslations(lang: string): Promise<Translations>;
+
+    /**
+     * Reduce the languages array provided by a package to an array of file paths of translations to load
+     * @param pkg  - The package data
+     * @param lang - The target language to filter on
+     * @returns An array of translation file paths
+     */
+    protected _filterLanguagePaths(pkg: ModuleData | SystemData, lang: string): string[];
+
+    /**
+     * Load a single translation file and return its contents as processed JSON
+     * @param src - The translation file path to load
+     * @returns The loaded translation dictionary
+     */
+    protected _loadTranslationFile(src: string): Promise<Translations>;
+
+    /**
+     * Return whether a certain string has a known translation defined.
+     * @param stringId - The string key being translated
+     * @param fallback - Allow fallback translations to count?
```

Comment:
maybe create a report about this being unused in foundry gitlab?

Reply:
https://gitlab.com/foundrynet/foundryvtt/-/issues/5608